__tests__/app.test.js

```diff
@@ -27,6 +27,30 @@ describe("api/categories", () => {
   });
 });
 
+describe("api/reviews", () => {
+  describe("GET", () => {
+    test("should respond with an array of objects with each object having a property of: owner, title, review_id, category, review_img_url, created_at, votes, designer, comment count sorted by date in DESC order", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviews.length > 0).toBe(true);
+          body.reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner");
+            expect(review).toHaveProperty("title");
+            expect(review).toHaveProperty("review_id");
+            expect(review).toHaveProperty("category");
+            expect(review).toHaveProperty("review_img_url");
+            expect(review).toHaveProperty("created_at");
+            expect(review).toHaveProperty("votes");
+            expect(review).toHaveProperty("designer");
+            expect(review).toHaveProperty("comment_count");
+          });
+        });
+    });
+  });
+});
```

Comment:
There is no test here to show that it is sorted by created_at in descending order. Look into using jest-sorted for this